src/main/java/games/strategy/engine/framework/ui/GameChooserModel.java

```diff
@@ -177,16 +174,9 @@ private static GameChooserEntry createEntry(final URI uri)
 
     final File parentFolder = mapFolder.exists() ? mapFolder : mapDir;
     final File games = new File(parentFolder, "games");
-
-    if (!games.exists()) {
-      // no games in this map dir
-      return entries;
-    }
-    if (games.listFiles() != null) {
-      for (final File game : games.listFiles()) {
-        if (game.isFile() && game.getName().toLowerCase().endsWith("xml")) {
-          createGameChooserEntry(game.toURI()).ifPresent(entries::add);
-        }
+    for (final File game : FileUtils.listFiles(games)) {
+      if (game.isFile() && game.getName().toLowerCase().endsWith("xml")) {
+        createGameChooserEntry(game.toURI()).ifPresent(entries::add);
```

Comment:
Not directly related to this PR, but I see a lot of "one time use" variables here in this method, that could be merged into a single line.
Also Streams to be used to create the set directly:
```java
return FileUtils.listFiles(games).stream()
  .filter(File::isFile)
  .filter(game -> game.getName().toLowerCase().endsWith("xml"))
  .map(File::toURI)
  .map(GameChooserModel::createGameChooserEntry)
  .filter(Optional::isPresent)
  .collect(Collectors.toSet());
```

Reply:
Agreed, but probably should be done in a separate PR.  I'm ok with the readability of the functional appearance of the stream equivalent, even though it is a bit long, but it may be too "state machine-ish" for some. :smile:  (BTW, there's a missing `.map(Optional::get)` before the `.collect()`.

Reply:
Right, but OK